### DISCUSSION TITLE
23.07 Main variant multiple outputs in Cubase?

### DISCUSSION THREAD

**Discussion by susalars** [DISTRHO/Cardinal#555] [Q&A]
I've read all the discussions on this topic here on GitHub. But I still don't know how (or is this even possible) to route multiple outputs from Cardinal Main to Cubase. 

In the current version (23.07), the Main variant is recognized as an insert VST3 plugin on an audio channel inside Cubase. So, I can't find a way to utilize all 8 outputs.

Is this because I have limited amount of physical outputs on my audio interface (Focusrite 6i6 Gen II, which only has 4 I/Os)? If not, is multiple outputs currently even possible in Cubase?

Thanks!

**Comment by falkTX** [comment]
Cardinal sets the first 2 IO as "main" and active, with the other 3x2 pairs without the main/active flag, so the host is free to enable it as needed.
How that ends up working differs between DAWs, best to wait for someone with experience in Cubase to pitch in

**Reply by susalars** [reply]
Thank you falkTX! ... Well, fingers crossed!

**Reply by protectorateindustrial** [reply]
Currently its not possible to utilise the additional outputs in Cubase in a standard stereo project. If the main variant would be designated as a synth all the audio i/o would be addressable in Cubase. That, or additional audio i/o in the synth variant.

**Reply by falkTX** [reply]
that is my plan, get the main variant to appear as "synth" (it is just a category change, no real changes) and add some optional sidechain ports to the FX variant for those that were using the main variant as FX chain.
that should cover pretty much all cases while not being very disruptive.

**Reply by protectorateindustrial** [reply]
Perfect! Thank you :)
